web/management/commands/create_test_data.py

```
@@ -549,3 +582,254 @@
             self.stdout.write(f"Created images for product: {product.name}")
 
         self.stdout.write(self.style.SUCCESS("Successfully created test data"))
+
+
+    def create_exams_and_quizzes(self, courses, sessions, students, teachers):
```

Ruff (0.8.2)

`create_exams_and_quizzes` is too complex (13 > 10)

(C901)